tools/deployment/osqueryctl

```diff
@@ -69,7 +72,8 @@ exec_with_env() {
 start() {
   check_config
   if [ $OS = "darwin" ]; then
-    launchctl start $PLIST_PATH
+    cp $PLIST_INSTALLATION_PATH $PLIST_PATH
+    launchctl load $PLIST_PATH
```

Comment:
Ah, I remember now! I wonder if a conditional makes sense here to check if the copied plist exists, if so, copy the "potentially new" plist on update. Let me reread some of that initial discussion and think about this.